openedx_webhooks/github/dispatcher/actions/test/test_github_cla.py

```diff
@@ -0,0 +1,139 @@
+"""
+Confirm expected behavior of the CLA-related functionality
+"""
+# Disable these warnings due to how the fixtures are constructed and used.
+# pylint: disable=redefined-outer-name,unused-argument
```

Comment:
Note that pylint doesn't run against this repo, so there are myriad violations.
This file, however, is clean.